packages/fast-components-react-msft/src/slider-label/README.md

```diff
@@ -1,5 +1,23 @@
-# Slider label
+## Slider label
+*Slider labesls* are styled label versions of base *Slider track item" components positioned at specific values on a slider's track to act as labels.  By default a *slider label* renders above the *slider*'s front and back bars and below the thumbs.
```

Comment:
- misspelt labels
- you have a " instead of a * on `*Slider track item*`

I'm also unclear about what front and back bars are.

Reply:
*Slider labels* are styled label versions of base *Slider track item* components positioned at specific values on a slider's track to act as labels.  By default a *slider label* renders above the *slider*'s track visual and below the thumbs.